kyo-prelude/shared/src/main/scala/kyo/kernel/Trace.scala

```diff
@@ -1,11 +1,12 @@
 package kyo.kernel
 
 import internal.*
+import kyo.Frame
 import scala.annotation.tailrec
 import scala.util.control.NonFatal
 import scala.util.control.NoStackTrace
 
-class Trace(
+final class Trace(
```

Comment:
I wonder if we could enforce that all classes are either final/abstract/sealed...